src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -137,6 +140,7 @@ public ClientDevicesAuthService(Topics topics, GroupManager groupManager, Certif
         SessionCreator.registerSessionFactory("mqtt", mqttSessionFactory);
         certificateManager.updateCertificatesConfiguration(new CertificatesConfig(this.getConfig()));
         sessionManager.setSessionConfig(new SessionConfig(this.getConfig()));
+        certificateManager.setCertificateStoreConfig(this.getConfig(), this.getRuntimeConfig());
```

Comment:
Take a look at how we deserialize GroupDefinition - ideally we can just follow a similar pattern here.

A couple benefits (I can think of):
* Don't need to plumb `Topics` all the way through. No need to deal with casting. We can add appropriate getters
* Config is immutable. Behavior is easier to reason about
* No need to add a bunch of static strings to this class and import them in other places